contracts/Stage2/BondingEvent.sol

```diff
@@ -204,6 +206,9 @@ contract BondingEvent is AccessControl {
     function _bond(address _user, uint256 _amountSeuro, uint256 _maturity, uint256 _rate) private onlyOperator {
         // information about the liquidity position after it has been successfully added
         AddedLiquidityResponse memory added = addLiquidity(_user, _amountSeuro);
+
+        emit Bonding(_user, _amountSeuro, _maturity, _rate);
```

Comment:
not sure we need this event if we have the `NewBond` one in **BondStorage** ... they happen in same transaction and the **BondStorage** one has more data

Reply:
that's fair, fixed 👍